test/e2e/authentication/login/login.page.ts

```diff
@@ -20,40 +23,33 @@ export class LoginPageObject {
 
   }
 
-  navigateToForgotPasswordPage() {
+  private getAllErrorMessages(): ElementArrayFinder {
```

Comment:
I would prefer to put private member functions on the bottom of the class. Might need to put a check on tslint for this tough..